app/views/pages/terms_conditions.html.erb

```diff
@@ -0,0 +1,181 @@
+      “<strong>us</strong>”). We are a central government department and have our registered office at
+      [REGISTERED OFFICE ADDRESS].
+    </p>
+
+    <p class="govuk-body">
+      You can contact us using the following email address:
+      <a href='mailto: [EMAIL ADDRESS]' class='govuk-link'>[EMAIL ADDRESS]</a>
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Access to the Site
+    </h3>
+
+    <p class="govuk-body">
+      We shall not be liable if for any reason the Site is unavailable at any time or for any period. From time to time,
+      we may restrict access to all or some parts of the Site to users who have registered with us.
+    </p>
+
+    <p class="govuk-body">
+      If you choose, or are provided with, a user identification code, password or any other piece of information as
+      part of our security procedures, you must treat such information as confidential, and you must not disclose it to
+      any third party.
+    </p>
+
+    <p class="govuk-body">
+      We reserve the right to restrict or deny you access to all or some part of the Site if in our opinion, you have
+      failed to comply with these General Terms.
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Hyperlinking to the Teachers: claim back your student loan repayments website
+    </h3>
+
+    <p class="govuk-body">
+      We do not object to you linking directly to pages on this site and you do not need to ask permission to do so.
+      However, we do not permit our pages to be loaded into frames on your site. The Teachers: claim back your student
+      loan repayments pages must be displayed in the user's entire browser window.
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Third Party Content and Hyperlinking from the Service
+    </h3>
+
+    <p class="govuk-body">
+      Where the Service contains links to other sites and resources, we are not liable or responsible for the content or
+      reliability of those websites and resources and do not necessarily endorse the views expressed within them.
+    </p>
+
+    <p class="govuk-body">
+      We aim to replace broken links to other sites but cannot guarantee that these links will always work as we have no
+      control over the availability of other sites.
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Virus protection
+    </h3>
+
+    <p class="govuk-body">
+      We make every effort to check and test material at all stages of production. It is always wise for you to run an
+      anti-virus program on all material downloaded from the Internet as we do not guarantee our service will be secure
+      or free from bugs or viruses. We cannot accept any responsibility for any loss, disruption or damage to your data
+      or your computer system which may occur whilst using material derived from this website.
+    </p>
+
+    <p class="govuk-body">
+      You must not misuse our service by knowingly introducing viruses, trojans, worms, logic bombs or other material
```

Comment:
This paragraph appears in the privacy policy too. Should it only be in one or the other?

Reply:
Good shout, but I think it would be better to have it in the T&Cs and remove from the privacy policy as to me it feels like the better place to have it.